infra/docker-compose.yaml

```diff
@@ -1,4 +1,42 @@
 services:
+  firefly-core:
+    image: ghcr.io/gytis-ivaskevicius/firefly-core:latest
```

Comment:
Are we okay with this? this was the easiest way to get image working in this repo

Reply:
yes